src/telemetryGenerator.node.ts

```diff
@@ -1,6 +1,11 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+// This line should always be right on top.
```

Comment:
Was this intended on being in?

Reply:
Yes, this is requried, beacuse i'm changing some of the code, and now this code ends up pulling a few of our extension code which relines on this being set. Our code only has 1 entry point, extension.ts 
this file is called from node.js which ends up pulling in some of our extension code, i.e. we have another entry point which pulls in other stuff and things can fall over when running this. Hope that makes sense.